test/test-diagnostics-channel-http-server-start.spec.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const test = require('tape');
+const common = require('./common.js');
+const { AsyncLocalStorage } = require('async_hooks');
+const dc = require('../dc-polyfill.js');
+const assert = require('assert');
+const http = require('http');
+
+const als = new AsyncLocalStorage();
+let context;
+
+test('test-diagnostics-channel-http-server-start', t => {
```

Comment:
This test has corresponding additions to the http server code in Node.js core to produce these events, so this won't pass on versions prior to that. We can probably just omit this test.

Reply:
Weird, might be a bug with the test runner then, seems like it's passing in all tested versions of node

Reply:
```js
if (MAJOR < 14) return;
if (MAJOR === 14 && MINOR < 17) return;
if (MAJOR === 15 && MINOR < 1) return;
```